src/components/ebay-snackbar-dialog/README.md

```diff
@@ -0,0 +1,27 @@
+<h1 style='display: flex; justify-content: space-between; align-items: center;'>
+    <span>
+        ebay-snackbar-dialog
+    </span>
+    <span style='font-weight: normal; font-size: medium; margin-bottom: -15px;'>
+        DS v1.0.0
+    </span>
+</h1>
+
+```marko
+<ebay-snackbar-dialog>
+    Basic Snackbar
+</ebay-snackbar-dialog>
+```
+
+## Description
+
+A snackbar is a non-modal dialog that appears in response to a lightweight user action. It disappears automatically after a minimum of 6 seconds.
+
+The user manages the state of the snackbar, and so should provide the open state as a boolean as well as a function to close the snackbar when the on-close event occurs.
```

Comment:
Second part of this sentence needs removing (regarding the close function requirement).

Reply:
After today's discussion, my interpretation is that if this component were a fully controlled component, then this sentence and instruction would be needed. However, given that it is not a fully controlled component, the sentence needs modifying to be something like:

`The application manages the initial state of the snackbar, and so should provide the open state as a boolean. The on-close event handler can be used to synchronize application state with the component's internal state`.

Reply:
okay I adjusted the README... let me know what you think